packages/app/src/components/Randomizer/Randomizer.css

```diff
@@ -29,6 +29,26 @@
     margin-top: 1em;
     margin-bottom: 0.5em;
 }
+.randomizer-button-group {
+    display: flex;
+    width: 360px;
```

Comment:
Should this be `min-width`? Now the buttons have effective max width of `360px` which causes them to wrap the text over multiple lines, even on desktop where there is plenty of room. By changing this to `min-width` it still allows rendering as a single line, expanding to up the `max-width`, given there is room.

I quickly experimented this within chrome dev tools and IMO it looks a lot better when the button text is rendered on a single line, even if the buttons have slightly different sizes then.